script.tubecast/resources/lib/tubecast/youtube/app.py

```diff
@@ -430,7 +430,7 @@ def report_state_change(self, status_code, current_time, duration):  # type: (in
     def __post_bind(self, sc, postdata):  # type: (str, dict) -> None
         self.ofs += 1
         post_data = {"count": "1", "ofs": str(self.ofs), "req0__sc": sc}
-        for key in postdata.keys():
+        for key in list(postdata.keys()):
```

Comment:
I don't think it is needed to use `list` here, as `.keys()` returns an iterator, which is needed in the `for`. But it won't break either.

Reply:
this was to silence 2to3 in addon checker, performance is likely lower and this is a false positive as I'm iterating (https://stackoverflow.com/questions/27476079/why-does-2to3-change-mydict-keys-to-listmydict-keys). I prefer to keep the list though, its not a big issue